include/constants/flags.h

```diff
@@ -41,12 +41,12 @@
 #define FLAG_TEMP_1F     (TEMP_FLAGS_START + 0x1F)
 #define TEMP_FLAGS_END   FLAG_TEMP_1F
 
-#define FLAG_UNUSED_0x020    0x20 // Unused Flag
-#define FLAG_UNUSED_0x021    0x21 // Unused Flag
-#define FLAG_UNUSED_0x022    0x22 // Unused Flag
-#define FLAG_UNUSED_0x023    0x23 // Unused Flag
-#define FLAG_UNUSED_0x024    0x24 // Unused Flag
-#define FLAG_UNUSED_0x025    0x25 // Unused Flag
+#define FLAG_UNUSED_0x020    0x20 // Unused Flag //Used for DEBUG functions, can be changed in "constants/battle_config.h"
+#define FLAG_UNUSED_0x021    0x21 // Unused Flag //Used for DEBUG functions, can be changed in "constants/battle_config.h"
+#define FLAG_UNUSED_0x022    0x22 // Unused Flag //Used for DEBUG functions, can be changed in "constants/battle_config.h"
+#define FLAG_UNUSED_0x023    0x23 // Unused Flag //Used for DEBUG functions, can be changed in "constants/battle_config.h"
+#define FLAG_UNUSED_0x024    0x24 // Unused Flag //Used for DEBUG functions, can be changed in "constants/battle_config.h"
+#define FLAG_UNUSED_0x025    0x25 // Unused Flag //Used for DEBUG functions, can be changed in "constants/battle_config.h"
```

Comment:
Unneeded, as by my comments in `include/constants/debug_config.h`, it should be disabled by default.